src/FireHydrantPhysicsComponent.cpp

```diff
@@ -77,13 +77,19 @@ void FireHydrantPhysicsComponent::updatePhysics(float deltaTime) {
 
          // Initialize fracture variables if the player hits the object hard enough
          if (objHit->velocity >= 10) {
+			// Do effects once on hit
+			if (!holder_->fracture) {
+				// Spawn billboard
+				holder_->spawnPlayerHitBillboardEffect(holder_->getPosition());
+
+				// Play sound effect.
+				AudioManager& audioManager = AudioManager::instance();
```

Comment:
If we want sure, it just plays every single time so it's a lot of overlap at once; will change back